src/wallet/rpcwallet.cpp

```diff
@@ -3818,6 +3833,186 @@ static UniValue listlabels(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue send(const JSONRPCRequest& request)
```

Comment:
Might use the non-deprecated constructor. E.g. #19528

Reply:
I switched to the fancy new thing.